components/content/inspira/ui/compare/StarField.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="star-field">
+    <div
+      v-for="star in stars"
+      :key="star.id"
+      :style="{
+        top: star.top,
+        left: star.left,
+        width: `${star.size}px`,
+        height: `${star.size}px`,
+        '--twinkle-duration': `${star.twinkleDuration}s`,
+        '--drift-duration': `${star.driftDuration}s`,
+        '--drift-direction': `${star.driftDirection}px`,
+        '--opacity-start': star.opacityStart,
+        '--opacity-end': star.opacityEnd,
+      }"
+      class="star"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+interface Props {
+  starsCount?: number;
+}
+
+interface Star {
+  id: number;
+  top: string;
+  left: string;
+  size: number;
+  twinkleDuration: number;
+  driftDuration: number;
+  driftDirection: number;
+  opacityStart: number;
+  opacityEnd: number;
+}
+
+const props = withDefaults(defineProps<Props>(), {
+  starsCount: 130,
+});
+
+function random(min: number, max: number): number {
+  return Math.random() * (max - min) + min;
+}
+
+function randomSize(): number {
+  // Randomly return either 1 or 2
+  return Math.random() < 0.5 ? 1 : 2;
+}
+
+const stars = computed(() =>
+  Array.from(
+    { length: props.starsCount },
+    (_, i): Star => ({
+      id: i,
+      top: `${random(0, 100)}%`,
+      left: `${random(0, 100)}%`,
+      size: randomSize(),
+      twinkleDuration: random(2, 4),
+      driftDuration: random(5, 10),
+      driftDirection: random(-50, 50),
+      opacityStart: random(0.1, 0.3),
+      opacityEnd: random(0.7, 1),
+    }),
+  ),
+);
+</script>
+
+<style scoped>
+.star-field {
```

Comment:
These properties and some of the other properties can directly be used as tailwind class. `absolute inset-0 overflow-hidden`

Reply:
I don't think the `v-bind` would help in this case because `v-bind` can only access component-level reactive properties, and cannot bind to individual elements generated within a v-for loop(as per my knowledge) or I am not getting your query properly.
For now, I've added the prefix `inspira` to variable names to avoid naming conflicts.